@here/harp-datasource-protocol/lib/TechniqueParams.ts

```diff
@@ -377,6 +377,13 @@ export interface StandardTechniqueParams extends BaseTechniqueParams {
      */
     alphaMap?: string | TextureBuffer;
     alphaMapProperties?: TextureProperties;
+
+    /**
+     * Whether this object can receive a shadow. Note, shadows must be enabled on the [[MapView]]
+     * for this to take affect. Note also that this also only receives shadows, a standard technique
+     * can't cast because it is flat.
+     */
+    enableShadows?: boolean;
```

Comment:
I would stick to the three.js name (i.e. castShadow and receiveShadow). This would allow us later to remove all hardcoded logic that maps technique properties to material or object properties

Reply:
Removed as discussed.